deps/libcypher-parser/lib/src/ast_call.c

```diff
@@ -26,6 +26,7 @@ struct call_clause
     const cypher_astnode_t *proc_name;
     const cypher_astnode_t **args;
     unsigned int nargs;
+    const cypher_astnode_t *predicate;
```

Comment:
Is that how libcypher-parser describes filters, as a sub tree of a given ast node?
I'm just used to all clauses being at the same AST tree level